code/client/src/pages/Create.jsx

```diff
@@ -106,6 +107,13 @@ const Create = () => {
       message.error('Cannot deploy wallet. Error: root is not set.')
       return
     }
+
+    // Ensure valid address for both 0x and one1 formats
+    const validLastResortAddress = util.validateAddress(lastResortAddress)
```

Comment:
The bech32 address which is deemed valid here must be converted to ETH address prior to processing below